src/deweb/redirection.ts

```diff
@@ -21,12 +28,12 @@ type DewebStatus = {
  *
  * If no provider is available, we redirect to the default URL
  * @param uri - The mns domain to resolve. It must be suffixed with a '.massa'
- * @param chainId - The returned url must belong to a provider having the same same chainid. If null, we don't check the chainId
+ * @param chainId - The returned url must belong to a provider having the same chainid. Mainnet by default
  * @returns The resolved URI
  */
 export async function resolveDeweb(
   uri: string,
-  chainId: number | null = null
+  chainId: bigint = CHAIN_ID.Mainnet
```

Comment:
no need to declare chainId type here ": bigint" it is inferred from the default value